### PR TITLE
Areas almayer qol

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
added some more areas to almayer
reworked lower engi areas position giving them parent for being lower etc...
remove an extra light

# Explain why it's good for the game
More areas allow to make it easier to navigate the ship and make the localisation system more efficient...
better parent avoid repeating lines of code and make it easier to maintain/change it in the future.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
fix: removed an extra light in bravo cryo room(to do just saw it)
maptweak: added more areas (stern_point_defense, Lower Deck Stern Hull, Upper Deck port Hallway Upper Deck starboard Hallway)
/:cl:

### CODE REVIEW

File: code/game/area/almayer.dm [hunk 1]
```diff
@@ -393,6 +390,19 @@
...
+
+/area/almayer/hallways/upper
+	fake_zlevel = 1 // upperdeck
+
+/area/almayer/hallways/upper/port
+	name = "\improper Upper Deck port Hallway"
+	icon_state = "port"
+
+/area/almayer/hallways/upper/starboard
+	name = "\improper Upper Deck starboard Hallway"
```
Comment (Nanu308) [inline code/game/area/almayer.dm:403]: Capitalize Starboard and Port to keep it similar in style to how it is currently.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
see comment

File: code/game/area/almayer.dm (hunk 1 above)
Comment (Huffie56) [reply code/game/area/almayer.dm]: done

**Review by Nanu308 — APPROVED** [review]
mapping lgtm

**Review by Drulikar — APPROVED** [review]
Not currently finding any issues with these changes.

**Pull request merged**